File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function calc(oper, num1, num2) {
    switch(oper) {
        case "+":
            return num1 + num2;
        case "-":
            return num1 - num2;
        case "*":
            return num1 * num2;
    }
}

function solution(binomial) {
    const str = binomial.split(" ");
    return calc(str[1], +str[0], +str[2]);
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    binomial = binomial.split(" ")
    op = binomial[1]
    num1 = int(binomial[0])
    num2 = int(binomial[2])
    
    if op == "+":
        return num1 + num2
    if op == "-":
        return num1 - num2
    if op == "*":
        return num1 * num2